docs/ext/commands/commands.rst

```diff
@@ -1176,3 +1176,72 @@ For example, to block all DMs we could do the following:
     Be careful on how you write your global checks, as it could also lock you out of your own bot.
 
 .. need a note on global check once here I think
+
+
+Hybrid Commands
+---------------
+
+.. versionadded:: 2.0
+
+:class:`.commands.HybridCommand` is a command that can be invoked as both a text and a slash command.
```

Comment:
Note that officially the library does not use "slash command" terminology, instead it uses "app commands" as a shortened form of "application command". In the user facing documentation it might be fine since there aren't any hybrid context menu commands so nothing might have to be done here.